tutorials/spherical_coordinates.md

```diff
@@ -68,6 +68,20 @@ that the coordinates were set correctly:
 
 @image html files/spherical_coordinates/inspector.png
 
+For loading Earth lunar DEMs, use the `MOON_SCS` surface tag:
```

Comment:
We should update the text above that says "he only surface model supported is WGS84".

Reply:
Done https://github.com/gazebosim/gz-sim/pull/1556/commits/c5ea0cb0204a46199f2c5be54da3361fb65f7b81